superlinter.yml

```diff
@@ -0,0 +1,25 @@
+name: Super-Linter
```

Comment:
I removed SuperLinter because it was annoying and it often didn't catch errors. This should probably be a separate PR.